pkg/web/templates/partials/secure_envelope/secure_envelope_list.html

```diff
@@ -1,6 +1,6 @@
 {{ if .DecryptedEnvelopes }}
```

Comment:
Should we check for the `DecryptedEnvelopes` in the list here or should I get the `SecureEnvelopes` list only, like what's currently happening on line 64 and then display the decrypted envelope in the `secure_envelope_detail`?

Reply:
I think we should get the secure envelopes only then display the decrypted envelope in detail. If we try to decrypt the entire list, then envelopes that can't be decrypted will be omitted and I think that's a poor UX for this particular view.